### PR TITLE
Tests for aborted downloads

### PR DESCRIPTION
**Pull request by PVince81**
This is a test to reproduce the issue https://github.com/owncloud/core/issues/22370 with PHP 7.

The only way to check if the issue occurs is by scraping the log file, so I improved that routine to download the file through the private API instead of copying. (copying failed on my envs due to different perms...)

Running this test is currently able to reproduce the issue after 10-15 iterations.

@DeepDiver1975 I think it would be fine to have a single executor for PHP 7 just for this one test, for now.

@jvillafanez @mrow4a please review

### CODE REVIEW

**Comment by PVince81** [comment]
Tested passed => this is wrong. Tests must fail on PHP 7

@DeepDiver1975 how are the PHP 7 smashbox tests going ?

**Comment by PVince81** [comment]
Please review @jvillafanez @DeepDiver1975 @VicDeo @mrow4a, even if tests pass now I think it's fine to merge them. Once the PHP 7 env is there it will reveal the failure and then maybe people will be more interested in reviewing the matching fixes.

File: python/smashbox/utilities/__init__.py [hunk 1]
```diff
@@ -577,46 +600,53 @@ def fatal_check(expr,message=""):
...
+    # download server log
+    log_url = 'http'
+    if config.oc_ssl_enabled:
+        log_url += 's'
+    log_url += '://' + config.oc_admin_user + ':' + config.oc_admin_password + '@' + config.oc_server
+    log_url += '/' + os.path.join(config.oc_root, 'index.php/settings/admin/log/download')
+
+    res = requests.get(log_url)
+
+    fatal_check(res.status_code == 200, 'Could not download the log file from the server, statuc code %i' % res.status_code)
```
Comment (VicDeo) [inline python/smashbox/utilities/__init__.py]: @PVince81 statuc

**Comment by VicDeo** [comment]
@PVince81 looks good except a typo :+1:

**Pull request merged**